src/Tests/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToPackAToolTargetingNonSupportedTFM.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.NET.Build.Tasks;
+using System;
+
+namespace Microsoft.NET.ToolPack.Tests
+{
+    public class GivenThatWeWantToPackAToolTargetingNonSupportedTFM : SdkTest
+    {
+
+        public GivenThatWeWantToPackAToolTargetingNonSupportedTFM(ITestOutputHelper log) : base(log)
+        {
+
+        }
+
+        [Theory]
+        // lower than netcoreapp2.0
+        [InlineData("TargetFramework", "netcoreapp2.0", "PackAsToolDoesNotSupportTFMLowerThanNetcoreapp21")]
+        [InlineData("TargetFramework", "netcoreapp1.1", "PackAsToolDoesNotSupportTFMLowerThanNetcoreapp21")]
+        [InlineData("TargetFrameworks", "netcoreapp2.0;netcoreapp2.1", "PackAsToolDoesNotSupportTFMLowerThanNetcoreapp21")]
+        // non netcoreapp
+        [InlineData("TargetFramework", "net46", "PackAsToolOnlySupportNetcoreapp")]
+        [InlineData("TargetFramework", "netstandard2.0", "PackAsToolOnlySupportNetcoreapp")]
```

Comment:
And I will explicit block full framework and netstarnard. They should not work in the beginning